docs/developers-guide.md

```diff
@@ -38,7 +38,7 @@ Test report will be available in HTML format in `fledge-gui/e2e-test-report/`; O
    ```
    $ sudo git clone https://github.com/fledge/fledge-gui.git
    $ sudo yarn
-   $ sudo yarn e2e --protractor-config=protractor_ci.conf.js
+   $ sudo yarn e2e
```

Comment:
You can add a note, 
 
For CI integration to run on Ubuntu machine, please use e2e/run script.